### PR TITLE
Add recipe for pyeasyfuse

### PR DESCRIPTION
**Pull request by priesgo**
This PR adds a recipe for the python package pyeasyfuse. This is a component of the fusion caller easyfuse, this is intended to be used from the main easyfuse workflow implemented in Nextflow.

It includes python dependencies and also some R dependencies on which it relies.

### CODE REVIEW

**Comment by priesgo** [comment]
@BiocondaBot please add label

**Comment by priesgo** [comment]
@bioconda/core would it be possible to add @ibn-salem as a member to bioconda to maintain this recipe and others from our team?

File: recipes/pyeasyfuse/meta.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - r-xml
+    - r-randomforest
+
+test:
+  commands:
+    - easy-fuse --help
+
+about:
+  home: "https://github.com/TRON-bioinformatics/easyfuse-src"
+  license: GPL-3.0
```
Comment (bgruening) [inline recipes/pyeasyfuse/meta.yml:53]: Can you please use a SPDF term

File: recipes/pyeasyfuse/meta.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  license_file: LICENSE
+  summary: "EasyFuse is a pipeline to detect fusion transcripts from RNA-seq data 
+  with high accuracy. The current version of EasyFuse uses two fusion gene 
+  detection tools, STAR-Fusion and Fusioncatcher along with a powerful read 
+  filtering strategy, stringent re-quantification of supporting reads and 
+  machine learning for highly accurate predictions."
+  doc_url: "https://github.com/tron-bioinformatics/easyfuse-src"
+  dev_url:
```
Comment (bgruening) [inline recipes/pyeasyfuse/meta.yml:61]: Can be removed.

**Review by bgruening — APPROVED** [review]
Thanks @priesgo

File: recipes/pyeasyfuse/meta.yml (hunk 1 above)
Comment (priesgo) [reply recipes/pyeasyfuse/meta.yml]: Done

File: recipes/pyeasyfuse/meta.yml (hunk 2 above)
Comment (priesgo) [reply recipes/pyeasyfuse/meta.yml]: Done

**Pull request merged**